lib/rubyplot/scripting/line/geometry.rb

```diff
@@ -0,0 +1,24 @@
+module Rubyplot
+  class LineGeometry < Rubyplot::ArtistGeometry
+    attr_accessor :reference_lines
+    attr_accessor :reference_line_default_color
+    attr_accessor :reference_line_default_width
+    attr_accessor :hide_dots
+    attr_accessor :hide_lines
+    attr_accessor :show_vertical_markers
+    attr_accessor :dot_style
+    def initialize(*)
```

Comment:
What does the asterisk mean here?

Reply:
Sorry, 
That was possibly added by an automated linting tool that I was using. 
I removed it.